plugins/modules/pnp_intent.py

```diff
@@ -945,22 +991,26 @@ def get_diff_deleted(self):
                     params={"id": device_id},
                 )
 
-                self.log(str(response))
+                self.log("Device details for the device deleted with \
```

Comment:
self.log("Device details for the deleted device with serial number '{0}': {1}".format(device["deviceInfo"]["serialNumber"], str(response)), "DEBUG")

Reply:
Resolved